solver/src/block.rs

```diff
@@ -80,6 +89,12 @@ impl From<Piece> for Block {
             &["###", "#..", "#.."],    // Pentomino, V
             &["#.", "##", "#.", "#."], // Pentomino, Y
             &["##.", ".#.", ".##"],    // Pentomino, Z
+            &["##", "##"],             // Tetromino, Square
+         	&["##", ".#", ".#"],       // Tetromino, L
```

Comment:
I see several format issues in the Rust files. So, can you run `cargo fmt` to fix them automatically? Also, can you run `cargo clippy` so you can check if there is a better implementation?

Reply:
Done and done.